src/Components/DraggableResizer/DraggableResizerHandle.tsx

```diff
@@ -1,16 +1,28 @@
 // Libraries
-import React, {forwardRef, CSSProperties} from 'react'
+import React, {forwardRef, CSSProperties, useState} from 'react'
 import classnames from 'classnames'
 
+// Components
+import {SquareButton} from '../Button/Composed/SquareButton'
+
 // Types
-import {Gradients, Orientation, StandardFunctionProps} from '../../Types'
+import {
+  Gradients,
+  IconFont,
+  Orientation,
+  StandardFunctionProps,
+} from '../../Types'
 
 // Constants
 import {getColorsFromGradient} from '../../Utils/colors'
 
 export interface DraggableResizerHandleProps extends StandardFunctionProps {
-  /** Expects a number between 0 - 1 */
-  position: number
+  /** Enables a 0.0 direction (Left/Up) Collapse button */
```

Comment:
Is the 0.0 a float?

Reply:
In this case, it will always be 0 or 1, but when dragged, the value can be anywhere between 0-1